cost-analyzer/templates/kubecost-rbac-secret-template.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if eq (include "rbacTeamsEnabled" .) "true" }}
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: kubecost-rbac-secret
```

Comment:
do people need to be able to specify an existing secret?

Reply:
I would say no, this is specifically for cross-pod access to the rbac token signing key.